.github/workflows/test.yml

```diff
@@ -11,7 +11,9 @@ jobs:
     strategy:
       matrix:
         version: [20, 21]
-    runs-on: ubuntu-latest
+        os: [ubuntu-latest, windows-latest]
+      fail-fast: false
```

Comment:
What does this do?

Reply:
`fail-fast: true` (the default) means that if any CI job fails, the rest are immediately cancelled. `fail-fast: false` lets them all run to completion. `true` saves on CI time, `false` leads to easier debugging.